src/shared/tpm2-util.c

```diff
@@ -2705,7 +2705,9 @@ int tpm2_get_best_pcr_bank(
 
                 /* If variable is not set use guesswork below */
                 log_debug("Boot loader didn't set LoaderTpm2ActivePcrBanks EFI variable, we have to guess used PCR banks.");
-        } else {
+        } else if (efi_banks == UINT32_MAX)
+                log_debug("Boot loader set LoaderTpm2ActivePcrBanks EFI variable to UINT32_MAX (i.e. GetActivePcrBanks() API is not available in the firmware), we have to guess used PCR banks.");
```

Comment:
I think this is too much of an implementation details to what value the variable was set. (And UINT32_MAX is a C construct, not necessarily clear to the reader.)

"Boot loader set LoaderTpm2ActivePcrBanks to indicate firmware doesn't implement GetActivePcrBanks, we have to guess available PCR banks" ?